melodyPortfolio.html

```diff
@@ -5,32 +5,117 @@
 		<title>Melody Sylvestre's portfolio</title>
 		<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
 		<link rel="stylesheet" type="text/css" href="style.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	</head>
 	
 	<body>
+		
 		<header> 
 			<h1>Melody Sylvestre</h1>
+			<nav>
+				<a href="#about_me">About me</a>
+				<a href="#portfolio">Portfolio</a>
+				<a href="#contact">Contact</a>
+			</nav>	
 		</header>
 		
 		<div class="introduction_section">
-			<h2>Welcome to my portfolio</h2>
-		</div>
+			<h2>Hi and welcome to my portfolio</h2>
+			<p>I'm Melody, a software developer based in Bristol.</p>
+		</div> 
 		
-		<div class="aboutMe_section">
+		<div id="about_me" class="aboutMe_section">
 			<h2>About me</h2>
-		</div>	
+			<div class="aboutMe_card">
+				<img src="images/me_crop.jpg" alt="A picture of me">
+				<div class="aboutMe_text">
+					<p>A nice little paragraph where I talk about myself. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla. Bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla</p>
+				 <a href="images/cv_msylvestre.pdf">Download my CV</a>
```

Comment:
consider having your CV open in a new tab?

Reply:
done!